### DISCUSSION TITLE
I want to multiply two field's value and want its result in 3rd field. How to do that?

### DISCUSSION THREAD

**Discussion by Abhilash989** [YetiForceCompany/YetiForceCRM#12198] [How to]
I have two fields like quantity and unit price and want the result in third field (say Sum) on mouse over or tab press.

**Comment by martin-crockett** [comment]
yes i would also like that, because right now i have a workflow that when you save it does the math but i would like it to do it right after you enter values.

**Comment by kassimg12** [comment]
Is this question answered? this is my question too. if anyone can help, please.

**Comment by martin-crockett** [comment]
> Is this question answered? this is my question too. if anyone can help, please.

not answered yet
